### DISCUSSION TITLE
Running on GCP Cloud Run or Scaleway Serverless Containers

### DISCUSSION THREAD

**Discussion by theobouwman** [ag2ai/faststream#1572] [Q&A]
Hello,

we are planning to deploy faststream workers on GCP Cloud Run or Scaleway Serverless Containers. We want to do this because continuous deployments are easy to setup and our current infrastructure is already setup on these services.

The only recommendation for GCP Cloud Run and Scaleway Serverless Containers is that the container exposes a HTTP port. We start faststream with `CMD faststream run worker.app:app` in the container and it does not expose any http port which we can use to check if the service is running. 

So how could we solve this issue? I assume others have solved this already? Thanks

**Comment by Lancetnik** [comment]
Hi! I think, your question is strongly related to #1181 and #1582 Issues
Now you can just run **FastStream** with any asgi framework in the same runtime to implement HTTP-probes manually: https://faststream.airt.ai/latest/getting-started/integrations/frameworks/
